Source/ThirdParty/skia/CMakeLists.txt

```
@@ -1,10 +1,18 @@
 set_property(DIRECTORY . PROPERTY FOLDER "skia")
 
+if (ANDROID)
+    set(is_android TRUE)
+endif ()
+
 # Skia dependencies not used directly in WebKit.
 find_package(Freetype 2.9.0 REQUIRED)
 find_package(Fontconfig 2.13.0 REQUIRED)
 find_package(WebP REQUIRED COMPONENTS mux)
 
+if (is_android)
+find_package(Expat REQUIRED)
+endif ()
```

Check `ANDROID` directly + indentation:

```suggestion
if (ANDROID)
    find_package(Expat REQUIRED)
endif ()
```

---

Fixed